yb-voyager/cmd/common.go

```diff
@@ -135,53 +135,34 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 	return tableNameVsFileNameMap
 }
 
-func UpdateTableRowCount(source *srcdb.Source, exportDir string, tablesProgressMetadata map[string]*utils.TableProgressMetadata) {
+func UpdateTableApproxRowCount(source *srcdb.Source, exportDir string, tablesProgressMetadata map[string]*utils.TableProgressMetadata) {
 	var maxTableLines, totalTableLines int64
 
 	payload := callhome.GetPayload(exportDir)
 
-	fmt.Println("calculating num of rows to export for each table...")
+	utils.PrintAndLog("calculating approx num of rows to export for each table...")
 	if !source.VerboseMode {
 		go utils.Wait()
 	}
```

Comment:
If we can now get row count very quickly, we should get rid of the spinner from this code-path.